### PR TITLE
Added a link back to http://www.code.org on studio.code.org header.

### PR DESCRIPTION
**Pull request by CaptainStack**
Includes styling identical to header links on code.org.

### CODE REVIEW

**Comment by CaptainStack** [comment]
link-back

**Comment by CaptainStack** [comment]
screen shot 2015-01-05 at 2 47 17 pm

File: dashboard/app/assets/stylesheets/application.css.scss [hunk 1]
```diff
@@ -539,6 +539,25 @@ html, body {
...
+  text-decoration: none;
+}
+
+.headerlink:hover, .headerlink:active {
+  color: white;
+  border-bottom: solid 4px rgb(255, 138, 0);
+  background-color: initial;
+}
+
+.headerlink {
```
Comment (laurelfan) [inline dashboard/app/assets/stylesheets/application.css.scss]: should we use text-transform: uppercase instead of making the actual text uppercase?

**Comment by rlhawk** [comment]
I prefer text-transform: uppercase, but we haven't done that for other nav links. That's something to address with CSS refactoring.

**Comment by CaptainStack** [comment]
I'm going to leave it as is for now so we can get them all together when we refactor the CSS.

**Pull request merged**